armada_jupyter/podspec.py

```diff
@@ -30,4 +33,29 @@ def create_podspec_object(podspec_dict):
                     for key, value in container["resources"]["requests"].items()
                 }
 
+    # check that length of containers is exactly 1
+    # if not, raise an error
+    if len(podspec_dict["containers"]) != 1:
+        raise ValueError(
+            "Only one container per pod is supported. Please check your podspec."
+        )
+
+    # serviceport = job.podspec.containers[0].ports[0].containerPort
```

Comment:
What's all this commented code?

Reply:
Sorry, leftovers from getting github copilot to help me out.